src/app_service/service/network/service.nim

```diff
@@ -89,6 +96,13 @@ proc getNetwork*(self: Service, chainId: int): NetworkDto =
     if chainId == net.chainId:
         return net
 
+proc getNetworkByChainId*(self: Service, chainId: int): NetworkDto =
```

Comment:
@Khushboo-dev-cpp this function doesn't include test/prod as param?
I mean for the same chainId you will always get prod network, never test network.

Reply:
@Khushboo-dev-cpp cool, thanks, I didn't know that, I've thought that for example arbitrum and arbitrum test network have the same chain id.

Reply:
Aah I can understand, but yes they are different chainIds fir arb and arb test. Ive addressed the other comments from you btw